src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    public sealed class ReportGeneratorSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the list of coverage reports that should be parsed.
+        /// </summary>
+        public ReportGeneratorReportType[] ReportTypes { get; set; }
```

Comment:
To be more consistent with rest of cake and support more types change to  `ICollection<ReportGeneratorReportType>` as Array inhertics from that interface it should be a minor tweak.